api/src/test/scala/hmda/api/http/InstitutionsHttpApiSpec.scala

```diff
@@ -73,9 +74,11 @@ class InstitutionsHttpApiSpec extends WordSpec with MustMatchers with ScalatestR
       }
       Get("/institutions/12345/filings/xxxx") ~> institutionsRoutes ~> check {
         status mustBe StatusCodes.NotFound
+        responseAs[ErrorResponse] mustBe ErrorResponse(404, "xxxx filing not found for institution 12345", "institutions/12345/filings/xxxx")
       }
       Get("/institutions/xxxxx/filings/2017") ~> institutionsRoutes ~> check {
         status mustBe StatusCodes.NotFound
+        responseAs[ErrorResponse] mustBe ErrorResponse(404, "2017 filing not found for institution xxxxx", "institutions/xxxxx/filings/2017")
```

Comment:
This would require adding more logic to the api which is not in the scope of this pr. It probably should be ticketed though.